docs/customservices.md

```diff
@@ -10,7 +10,6 @@ within Homer.
 
 If you experiencing any issue, please have a look to the [troubleshooting](troubleshooting.md) page.
 
-
```

Comment:
Small nit: undo unrelated change. Here and lines 82 & 135

Reply:
That was my markdown linter, sorry :)